### DISCUSSION TITLE
HOW MUCH IS YOUR IBAX POINT? AND HOW WE INCREASE?

### DISCUSSION THREAD

**Discussion by metinx33** [IBAX-io/go-ibax#1220] [Q&A]
My ibax score is 25300. I have completed all the tasks including github except the recommendation tasks. What else should we do to get a higher score?

**Comment by harunkursun23** [comment]
SORU CEVAP ETKİNLİKLERİNE KATILMALISIN

**Comment by erhanqwerty** [comment]
ortalama 26000 30000 aralığında olabilir

**Reply by metinx33** [reply]
seninki benden daha fazlaymış.

**Reply by erhanqwerty** [reply]
25900

**Reply by metinx33** [reply]
bi ben zügürt kalmışım desene

**Comment by sedat2855** [comment]
20 000- 26 000
